src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -426,3 +426,83 @@
         - name: Create a vnet rule by providing the vnet and subnet name. The subnet id is created by taking the resource group name and subscription id of the SQL server.
           text: az sql server vnet-rule create --subnet subnetName --vnet-name vnetName
     """
+helps['sql managed-instance'] = """
```

Comment:
Would you consider abbreviating to `sql mi`? `managed-instance` is pretty long to type. Similar for `sql mdb`

Reply:
I have no strong preferences here. sql mi looks ok. "sql mdb" for me sound a bit strange and confusing. What do you say about "sql midb"?